src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -64,9 +66,16 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
     }
 
     renderButtons(paypalCheckoutInstance: PaypalClientInstance) {
-        const { paypalOptions, paymentMethod, container } = this._renderButtonsData as RenderButtonsData;
+        const { paypalOptions, paymentMethod, container, messagingContainerId } = this._renderButtonsData as RenderButtonsData;
         const { paypal } = this._window;
+        const state = this._store.getState();
+        const cart = state.cart.getCartOrThrow();
```

Comment:
I think it will be better to add `const config = state.config.getStoreConfig();` here and add some changes to line 75

Reply:
fixed